### DISCUSSION TITLE
History and Compression configuration changes.

### DISCUSSION THREAD

**Discussion by peregineme** [sysstat/sysstat#382] [Q&A]
When making changes to `/etc/cron.d/sysstat` concerning the intervals and counts for `sa1`, I see that these are effective once I close the file.
However, I see that when I did a change in `/etc/sysconfig/sysstat` to set `COMPRESSAFTER=1`, the files older than one day were not compressed, nor were new `sa` files created the next day compressed when they were one day old.
Is there some command we need to run to update the configuration?
In the Documentation, there is information on how to install the package, but not on how to make configuration changes.

**Comment by sysstat** [comment]
`/etc/sysconfig/sysstat` file is read by `sa1` and `sa2` scripts at run-time:
```
# Read configuration file, overriding variables set above
[ -r ${SYSCONFIG_DIR}/${SYSCONFIG_FILE} ] && . ${SYSCONFIG_DIR}/${SYSCONFIG_FILE}
```
So nothing should be done to make the change effective. Just edit the configuration file, save it (and make sure it is world readable).

**Reply by peregineme** [reply]
It seems both the `COMPRESSAFTER` and `HISTORY` actions are executed by `sa2` for both `sa` and `sar` files.
`sa2` was disabled in `/etc/cron.d/sysstat`, but once I enabled it, the history cleanup and file removal happened.
